allennlp/data/token_indexers/pretrained_transformer_indexer.py

```diff
@@ -28,7 +28,7 @@ class PretrainedTransformerIndexer(TokenIndexer):
 
     model_name : `str`
         The name of the `transformers` model to use.
-    namespace : `str`, optional (default=`tags`)
+    namespace : `str`, optional (default=`from_transformers`)
         We will add the tokens in the pytorch_transformer vocabulary to this vocabulary namespace.
         We use a somewhat confusing default value of `tags` so that we do not add padding or UNK
```

Comment:
```suggestion
        We use `from_transformers` as the default namespace, which gets special treatment in our
        `Vocabulary` class to not get padding or UNK tokens.
```

Actually, how are `UNK` and padding handled here?  Is it possible to set them using the transformer?  We probably *do* want UNK handling and padding tokens, we just need them set in the right way, both here and during loading.  This might need some additional changes to how we save and load vocabularies.

And, while we're at it, we probably want the padding and unk tokens to be namespace-dependent.